node/src/env0-deploy-utils.js

```diff
@@ -136,6 +150,10 @@ class DeployUtils {
     do {
       ({ status } = await this.apiClient.callApi('get', `environments/${environmentId}`));
 
+      if (status === 'WAITING_FOR_USER') {
+        throw new Error('Deployment is waiting for an approval. Run \'env0 approve\' or \'env0 cancel\' to continue.');
+      }
+
       if (environmentValidStatuses.includes(status)) return;
```

Comment:
Add CANCELED to `environmentValidStatuses`

Reply:
Also, I have already merged the new ABORTED status, so I recommend adding it as well

Reply:
e721896 e291656